types/is-expression/package.json

```diff
@@ -0,0 +1,20 @@
+{
+    "private": true,
+    "name": "@types/is-expression",
+    "version": "4.0.9999",
+    "projects": [
+        "https://github.com/pugjs/is-expression#readme"
+    ],
+    "dependencies": {
+        "acorn": "7.1"
```

Comment:
it's not the same as in upstream (only 7.1 pass the check).
https://github.com/pugjs/is-expression/blob/51045ecb52ad291d9df093c332debf28f64b726e/package.json#L13C16-L13C19